tests/playwright/src/specs/volume-smoke.spec.ts

```diff
@@ -62,16 +68,84 @@ describe('Volume workflow verification', async () => {
     await playExpect.poll(async () => await volumesPage.waitForVolumeExists(volumeName)).toBeTruthy();
   });
 
+  test('Delete volume from Volumes page', async () => {
+    let volumesPage = await navBar.openVolumes();
+    await playExpect(volumesPage.heading).toBeVisible();
+
+    let volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    playExpect(volumeRow).not.toBeUndefined();
+
+    volumesPage = await volumesPage.deleteVolume(volumeName);
+
+    await playExpect.poll(async () => await volumesPage.waitForVolumeDelete(volumeName)).toBeTruthy();
+    volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    playExpect(volumeRow).toBeUndefined();
+  });
+
   test('Delete volume through details page', async () => {
+    //re-create a new volume
     let volumesPage = await navBar.openVolumes();
     await playExpect(volumesPage.heading).toBeVisible();
 
-    const volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    const createVolumePage = await volumesPage.openCreateVolumePage(volumeName);
+    volumesPage = await createVolumePage.createVolume(volumeName);
+
+    await playExpect.poll(async () => await volumesPage.waitForVolumeExists(volumeName)).toBeTruthy();
+
+    //delete it from the details page
+    volumesPage = await navBar.openVolumes();
+    await playExpect(volumesPage.heading).toBeVisible();
+
+    let volumeRow = await volumesPage.getVolumeRowByName(volumeName);
     playExpect(volumeRow).not.toBeUndefined();
 
     const volumeDetails = await volumesPage.openVolumeDetails(volumeName);
     volumesPage = await volumeDetails.deleteVolume();
 
     await playExpect.poll(async () => await volumesPage.waitForVolumeDelete(volumeName)).toBeTruthy();
+    volumeRow = await volumesPage.getVolumeRowByName(volumeName);
+    playExpect(volumeRow).toBeUndefined();
+  });
+
+  test('Create volumes from bootc-image-builder', async () => {
+    //pull image from quay.io/centos-bootc/bootc-image-builder
+    const navigationBar = new NavigationBar(page);
+    let images = await navigationBar.openImages();
+    const pullImagePage = await images.openPullImage();
+    images = await pullImagePage.pullImage(imageToPull, imageTag);
+    await playExpect.poll(async () => await images.waitForImageExists(imageToPull)).toBeTruthy();
+
+    //count the number of existing volumes
+    let volumesPage = await navigationBar.openVolumes();
+    const previousVolumes = await volumesPage.countRowsFromTable();
```

Comment:
If you do this step before pulling the image you save up on navigating to and from the Image page to count the volumes.